### ISSUE TITLE
Is TissueMNIST the only dataset that uses a validation set?

### ISSUE THREAD

**Issue by ppsmk388** [microsoft/Semi-supervised-learning#134]
Thank you very much for your work, I would like to ask that is TissueMNIST the only dataset that uses a validation set?

**Comment by Hhhhhhao** [comment]
We do split a small part of training data as validation set when tuning the parameters. After deciding the parameters, the split validation set is put back into training. 

If a dataset originally has a validation set, we use that for tuning and do not put them to training data.

**Comment by caiolang** [comment]
Hello @Hhhhhhao , thank you for your answer, could you point out where this is done in the code? Thank you!
